### ISSUE TITLE
Move repository

### ISSUE THREAD

**Issue by pospi** [django-rea/rea-app#77]
I think @ivanminutillo's umbrella project and fork located at https://github.com/opencooperativeecosystem/rea-app is a better home for a React frontend app than sitting in the org with all the other django / Python stuff. How do we organise an ownership transfer?

**Comment by fosterlynn** [comment]
Agree.  But I doubt this repo is active any more.  Let's see what versions @ivanminutillo has going in various places, maybe he an provide an overview of where everything is now. :)
